README.md

```diff
@@ -4,13 +4,13 @@
 	</a>
 	<h3 align="center">Carbon Charts</h3>
 	<p align="center">
-		A reusable framework-agnostic D3 charting library.
+		A component library of 26 charts for [vanilla JavaScript](./packages/core), [Svelte](./packages/svelte), [React](./packages/react), [Vue.js](./packages/vue) and [Angular](./packages/angular).
```

Comment:
d3 is no longer a peer dependency so including this adds nothing. Also, got rid of the packages section by putting direct links to them here at the top.